src/Saas.Identity/Saas.Permissions/Saas.Permissions.Service/appsettings.json

```diff
@@ -5,20 +5,17 @@
     "Url": ""
   },
   "AzureAdB2C": {
-    "ClientId": "",
-    "ClientSecret": "",
-    "TenantId": "",
-    "IssuerDomain": ""
+    "SignedOutCallbackPath": "/signout/B2C_1A_SIGNUP_SIGNIN",
+    "SignUpSignInPolicyId": "B2C_1A_SIGNUP_SIGNIN"
   },
   "Logging": {
     "LogLevel": {
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
   },
-
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "PermissionsContext": "Server=(localdb)\\mssqllocaldb;Database=Saas.Permissions.Sql;Trusted_Connection=True;MultipleActiveResultSets=true"
+    "TenantsContext": "Server=(localdb)\\mssqllocaldb;Database=TenantsContext;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
This isn't right, this is a separate database from the tenants database. See this line: https://github.com/Azure/azure-saas/blob/main/src/Saas.Identity/Saas.Permissions/Saas.Permissions.Service/Program.cs#L41